dusty/constants.py

```diff
@@ -27,12 +25,17 @@
 DUSTY_GITHUB_PATH = 'gamechanger/dusty'
 DUSTY_BINARY_NAME = 'dusty'
 
-NGINX_MAX_FILE_SIZE = "500M"
+VM_IP_FROM_DOCKER = '172.17.42.1'
```

Comment:
This should always be the IP of the VM within the Docker bridge network. I think.